create_infotexts.py

```diff
@@ -387,7 +368,23 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
                 self.content_cats.append(places_mapping[place_string]["commonscat"])
 
         else:
-            place_as_wikitext += place_string
+            place_matches = []
+            for place in places_mapping:
+                if place.lower() in desc_string.lower():
+                    if places_mapping[place]["wikidata"]:
```

Comment:
You are still handling this as though multiple entries are expected.

Simply use something like the below (from L375, so all i indented once)
```python
place_matches = []
for place in places_mapping:
    if place.lower() in desc_string.lower():
        place_matches.append(place)

if len(place_matches) == 1:
    if places_mapping[place]["wikidata"]:
        place_as_wikitext = {{{{city|1={wikidata}}}}}".format(wikidata=places_mapping[place]["wikidata"]))
    else:
        place_as_wikitext = place

    if places_mapping[place].get('commonscat'):
        self.content_cats.append(places_mapping[place]["commonscat"])
else:
   self.meta_cats.append("Media_contributed_by_SMVK_without_mapped_place_value"

    if len(place_matches) > 1:
        print("Found {} places in description! -> {} img: {}".format(len(place_matches), place_matches, self.idno))
```